source-mysql-batch/driver.go

```diff
@@ -454,13 +457,18 @@ func (c *capture) worker(ctx context.Context, bindingIndex int, res *Resource) e
 		return fmt.Errorf("error parsing template: %w", err)
 	}
 
-	var pollInterval = 24 * time.Hour
+	// Polling interval can be configured per binding. If unset, falls back to the
+	// connector global polling interval. If that's also unset, falls back to 24h.
+	var pollStr = "24h"
```

Comment:
nit: this is not required / is duplicated given SetDefaults()

Reply:
Ah, good catch. I'll clean that up in a followup so I can click the merge button that much sooner here.